src/screens/LinkProject/LinkingProjectFailed.stories.ts

```diff
@@ -10,6 +13,9 @@ const meta = {
     projectId: "Project:abc123",
     configFile: "chromatic.config.json",
   },
+  decorators: [
+    storyWrapper(AuthProvider, () => ({ value: { accessToken: "token", setAccessToken: fn() } })),
+  ],
```

Comment:
Is this used everywhere? You would get more reuse by making it a global decorator in `.storybook/preview.tsx`

Reply:
Good call.